### ISSUE TITLE
accessibilityInformation should be a string

### ISSUE THREAD

**Issue by nickevansuk** [openactive/activation#165]
As shown on https://github.com/openactive/activation/issues/104, accessibilityInformation should be a string.

We'd previously discussed using a standard string for the case where this is `true`, see below:

Not like this:

```javascript
{
  "@context": "https://www.openactive.io/ns/oa.jsonld",
  "type": "Event",
  "accessibilityInformation": false,
}
```

Like this:

```javascript
{
  "@context": "https://www.openactive.io/ns/oa.jsonld",
  "type": "Event",
  "accessibilityInformation": "This route has been British Cycling assessed as an accessible route, meaning it is suitable for the majority of adaptive bikes. The route will have no or low levels of traffic, there will be plenty of space and will have a good surface throughout. If you have any questions about using this route on an adaptive bike on this ride, please call 0161 274 2000 and ask for the Recreation team.", /* Include this string (from the "Let's Ride" website), if the ride is accessible */
}
```
